addons/mass_mailing_sms/i18n/mass_mailing_sms.pot

```diff
@@ -207,7 +207,7 @@ msgstr ""
 #. module: mass_mailing_sms
 #: model:mailing.mailing,name:mass_mailing_sms.mailing_sms_1
 #: model:utm.source,name:mass_mailing_sms.mailing_sms_1_utm_source
-msgid "Extra Promo 2019-12-05 08:52:32"
+msgid "Extra Promo 2020-04-27 09:15:11"
```

Comment:
@qmo-odoo @tde-banana-odoo this kind of sucks, introduced at 1c614e34dc4d8fef95970aa781f56f4f0c532f1e

Reply:
Hahaha

Reply:
As name is generated we could simply set name on demo data heing.

Reply:
Sure, that would fix this one but is it a good idea to happend the raw data like this in the source name? don't we want a bit more context like "(created at ....)", human readable date, etc.
Or the question is why is it transaltable? Is the source `name` displayed somewhere to users?

Reply:
It is quite technical and mostly hidden but yes displayed to the user in UTM. However here we could simply use the date instead of a datetime, indeed could be improved a bit. We have to keep it short also, purpose is not to generate too long names.

About translation, well you can use it in website_links to generate short links. Translation makes sense.